src/distances/additiveSymmetric.ts

```diff
@@ -0,0 +1,18 @@
+import { NumberArray } from 'cheminfo-types';
+/**
+ *Returns the Additive Symmetric distance between vectors a and b
+ * @link [Additive Symmetric algorithm](https://www.naun.org/main/NAUN/ijmmas/mmmas-49.pdf)
+ * @param {NumberArray} a - first vector
+ * @param {NumberArray} b - second vector
+ * @returns {number}
```

Comment:
You can remove the type from the `@param` and completely remove the `@return` if there is no description. Just keep
```suggestion
 * @param a - first vector
```

You can do this across the whole project with a search / replace (hint: use a regexp)